src/net/fe/network/serverui/FEServerRunPanel.java

```diff
@@ -48,56 +42,53 @@ public FEServerRunPanel(int port) {
 		setLayout(new BorderLayout(0, 0));
 		mainPanel = new JPanel();
 		add(mainPanel, BorderLayout.CENTER);
-		
+
 		mainPanel.setLayout(new BoxLayout(mainPanel, BoxLayout.Y_AXIS));
-		
+
 		JLabel label = new JLabel();
 		label.setHorizontalAlignment(SwingConstants.CENTER);
 		label.setText("TODO");
 		label.setFont(getFont().deriveFont(20f));
 		//label.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
-		
+
 		try {
 			ip = InetAddress.getLocalHost().getHostAddress();
 			String text = "Server IP:" + ip;
-			if(port != FEServer.DEFAULT_PORT)
+			if (port != FEServer.DEFAULT_PORT)
 				text += ":" + port;
 			label.setText(text);
 		} catch (UnknownHostException e) {
 			label.setText("Error occured while getting the IP");
 		}
-		
+
 		mainPanel.add(label, BorderLayout.NORTH);
-		
+
 		Component verticalStrut = Box.createVerticalStrut(20);
 		mainPanel.add(verticalStrut);
-		
+
 		JPanel panel = new JPanel();
 		add(panel, BorderLayout.SOUTH);
 		panel.setLayout(new GridLayout(0, 1, 0, 0));
-		
+
 		JButton kickAll = new JButton("Kick all");
 		kickAll.addActionListener((e2) -> FEServer.resetToLobbyAndKickPlayers());
 		panel.add(kickAll);
-		
+
 		JButton copyToClipboard = new JButton("Copy IP to clipboard");
-		copyToClipboard.addActionListener(new ActionListener() {
-			public void actionPerformed(ActionEvent e) {
-				if(ip == null)
-					return;
-				String text = ip;
-				if(port != FEServer.DEFAULT_PORT)
-					text += ":" + port;
-				Toolkit.getDefaultToolkit().getSystemClipboard().setContents(new StringSelection(text), null);
-				
-			}
+		copyToClipboard.addActionListener(e -> {
+			if (ip == null)
+				return;
+			String text = ip;
+			if (port != FEServer.DEFAULT_PORT)
+				text += ":" + port;
+			Toolkit.getDefaultToolkit().getSystemClipboard().setContents(new StringSelection(text), null);
+
 		});
 		panel.add(copyToClipboard);
-		
+
 	}
-	
+
 	public void setPort(int port) {
```

Comment:
If you're going to remove the `port` variable, you might as well remove its setter too.